mmcv/ops/focal_loss.py

```diff
@@ -143,7 +143,8 @@ def forward(ctx,
                 weight: Optional[torch.Tensor] = None,
                 reduction='mean') -> torch.Tensor:
 
-        assert isinstance(target, (torch.LongTensor, torch.cuda.LongTensor))
+        assert isinstance(
```

Comment:
This assert is not necessary if torch.Tensor is valid

Reply:
On npu, target's class is torch.npu.LongTensor, which can't pass assert

Whether it can be modified to dtype judgment here, for example
```
assert isinstance(target.dtype, torch.long)
```

Reply:
Sure assert dtype seems reasonable.

Reply:
OK，did it